README.md

```diff
@@ -20,6 +20,8 @@ Currently this supports the following On-Demand and Spot pricing for the followi
  * `aws_elb`
  * `aws_lb`
  * `aws_nat_gateway`
+ * `aws_rds_cluster`
+ * `aws_rds_cluster_instance`
  * `aws_ecs_service` (Fargate On-Demand only)
```

Comment:
nit pick: should we sort this list alphabetically so it's easier to look for the thing you use?

Reply:
great idea 👍

Reply:
done